packages/client/src/store/CallState.ts

```diff
@@ -117,6 +117,7 @@ export class CallState {
   private callStatsReportSubject = new BehaviorSubject<
     CallStatsReport | undefined
   >(undefined);
+  private durationSubject = new BehaviorSubject<number>(0);
```

Comment:
@oliverlaz Please review [this commit](https://github.com/GetStream/stream-video-js/pull/1519/commits/98e742eb3e90412015f458967c2685f0a387b0da) where I introduced the call duration observable in the call state per your suggestion.

Reply:
hey @kristian-mkd, can you extract this change in a separate PR? We can ship it independently from the redesign project.

Reply:
Sure, moved to this PR https://github.com/GetStream/stream-video-js/pull/1528